components/edit-collective/sections/PaymentMethods.js

```diff
@@ -23,171 +24,142 @@ import {
   addCreditCardMutation,
   confirmCreditCardMutation,
 } from '../../recurring-contributions/UpdatePaymentMethodPopUp';
-import { withStripeLoader } from '../../StripeProvider';
 import StyledButton from '../../StyledButton';
 import { P, Span } from '../../Text';
 import EditPaymentMethod from '../EditPaymentMethod';
 import SettingsTitle from '../SettingsTitle';
 
-class EditPaymentMethods extends React.Component {
-  static propTypes = {
-    collectiveSlug: PropTypes.string.isRequired,
-    /** From graphql query */
-    data: PropTypes.object.isRequired,
-    /** From intl */
-    intl: PropTypes.object.isRequired,
-    /** From graphql query */
-    createCreditCardEditCollective: PropTypes.func.isRequired,
-    /** From graphql query */
-    confirmCreditCardEditCollective: PropTypes.func.isRequired,
-    /** From graphql query */
-    removePaymentMethod: PropTypes.func.isRequired,
-    /** From graphql query */
-    updatePaymentMethod: PropTypes.func.isRequired,
-    /** From graphql query */
-    editCollective: PropTypes.func.isRequired,
-    /** From stripeLoader */
-    loadStripe: PropTypes.func.isRequired,
-  };
-
-  constructor(props) {
-    super(props);
-    this.messages = defineMessages({
-      removeConfirm: {
-        id: 'paymentMethods.removeConfirm',
-        defaultMessage: 'Do you really want to remove this payment method?',
-      },
-    });
-  }
+function EditPaymentMethods(props) {
```

Comment:
Same here, let's keep it simple. Class components are fine.

Reply:
Addressed in https://github.com/opencollective/opencollective-frontend/pull/6193 😄